### PR TITLE
Changes in wizard to enable hide of previous and cancel button

### PR DESCRIPTION
**Pull request by akate1**
This PR has taken care of following things.
-     Allow wizard to close at any/each step. Also add story for the same
-     Make css of wizard button configurable
-     Hide previous button

image

### CODE REVIEW

File: src/wizards/Wizard.tsx [hunk 1]
```diff
@@ -46,12 +82,18 @@ type WizardProps = {
...
     onPrevious?: Function;
+    previousButtonClassName?: string;
     nextButtonText?: string;
     onNext?: Function;
+    nextButtonClassName?: string;
     finishButtonText?: string;
     onFinish?: Function;
+    finishButtonClassName?: string;
     cancelButtonText?: string;
+    showCancleButton?: boolean;
```
Comment (psivakumaran) [inline src/wizards/Wizard.tsx]: showCancleButton -> showCancelButton

Comment (akate1) [reply src/wizards/Wizard.tsx]: Done

File: src/wizards/Wizard.tsx [hunk 2]
```diff
@@ -34,6 +42,34 @@ type WizardStepNavDetails = {
...
+ * @param {previousButtonClassName} extranla CSS for previous button
+ * @param {nextButtonText} custom text for next button
+ * @param {onNext} callback function to call on click of next button
+ * @param {nextButtonClassName} extranla CSS for next button
+ * @param {finishButtonText} custom text for finish button
+ * @param {onFinish} callback function to call on click of finsih button
+ * @param {finishButtonClassName} extranla CSS for finish button
+ * @param {cancelButtonText} custom text for cancel button
+ * @param {showCancelButton} if true show cancel button on wizard else hide
+ * @param {cancleButtonClassName} extranla CSS for cancel button
```
Comment (psivakumaran) [inline src/wizards/Wizard.tsx]: Please verify its 'cancel' where required

Comment (akate1) [reply src/wizards/Wizard.tsx]: done

**Pull request merged**